tensorboard/examples/embedding_projector.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow as tf
+from tensorboard.plugins import projector
+import tensorflow_datasets as tfds
+
+# This example extends on the word embeddings tutorial found
```

Comment:
Conventionally we actually call these things "demos" and put them under the plugin directory, e.g. tensorboard/plugins/scalar/scalars_demo.py - I think that's probably best for consistency.

So this would go at tensorboard/plugins/projector/projector_demo.py.

Also, for the output directory, other demos use `/tmp/<plugin-name>_demo` so in this case that would be `/tmp/projector_demo` for consistency - that way it's easy to lump all the demo data together into a single directory that showcases multiple plugins.